Airports/EGCC/SMR/Geo.txt

```diff
@@ -72,7 +72,7 @@ N053.20.58.323 W002.16.49.089 N053.20.59.755 W002.16.46.107 smrCCrwhpts ;BZ1
 N053.21.12.829 W002.16.20.795 N053.21.13.244 W002.16.17.425 smrCCrwhpts ;DZ1
 N053.21.17.044 W002.16.10.364 N053.21.18.367 W002.16.07.647 smrCCrwhpts ;FZ1
 N053.21.22.316 W002.16.00.221 N053.21.22.690 W002.15.56.969 smrCCrwhpts ;HZ1
-N053.21.04.966 W002.16.22.198 N053.21.04.789 W002.16.19.941 smrCCrwhpts ;T1
+N053.21.04.966 W002.16.22.198 N053.21.04.789 W002.16.19.941 smrCCrwhpts ;T1l
```

Comment:
Why is this being renamed? Doesn't look like it's being changed on the chart or change description, and you're not editting the label.

Reply:
I must have typed it in without knowing uhh sorry....